src/FEValues/common_values.jl

```diff
@@ -139,20 +169,36 @@ function function_value(fe_v::AbstractValues, q_point::Int, u::AbstractVector, d
     end
     return val
 end
+function function_value(iv::InterfaceValues, q_point::Int, u::AbstractVector, dof_range = eachindex(u); use_element_a::Bool = true)
```

Comment:
Would it make sense to not collect all values in `u`, but to have two of them? That would make dof_range usage just like in the standard case
```
ic = InterfaceCache(...)

# x is the solution vector
xe_a = x[ic.dofs_a]
xe_b = x[ic.dofs_b]

u_dofs = dof_range(dh, :u) # Assuming single subdofhandler

# Alternative 1
function_value(iv, q_point, u_a, u_dofs; use_element_a = true)
# Alternative 2
function_value(iv.a, q_point, u_a, u_dofs)
```

But for jumps I suppose it then has to be
```
function_jump(iv, q_point, xe_a, xe_b, range)
function_jump(iv, q_point, xe_a, xe_b, range_a, range_b)
```
not too bad?